components/browser/engine-gecko-nightly/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt

```diff
@@ -60,24 +56,48 @@ import org.robolectric.RobolectricTestRunner
 @RunWith(RobolectricTestRunner::class)
 class GeckoEngineSessionTest {
 
+    private val navigationDelegate = ArgumentCaptor.forClass(
```

Comment:
nit: I would `lateinit` these and initialize in `setup`. Makes it less likely we'll see some test ordering issues / sideeffects in the future.

Reply:
TIL `lateinit` (this is my first kt commit :)) fixed thanks!